### PR TITLE
morenutrition.de

### PR DESCRIPTION
**Pull request by jellizaveta**
https://github.com/AdguardTeam/AdguardFilters/issues/124957

<details><summary>Screenshot:</summary>

Снимок экрана 2022-07-25 в 15 04 49

</details><br/>

### CODE REVIEW

File: AnnoyancesFilter/Cookies/sections/cookies_specific.txt [hunk 1]
```diff
@@ -145,6 +145,7 @@ whats-on-netflix.com#$#html.sp-message-open body { overflow-y: visible !importan
 !
 !
 !
+morenutrition.de###usercentrics-button
```
Comment (zloyden) [inline AnnoyancesFilter/Cookies/sections/cookies_specific.txt:148]: Where do you need this?

Comment (BlazDT) [reply AnnoyancesFilter/Cookies/sections/cookies_specific.txt:148]: Also not seeing it visible on the page

Comment (jellizaveta) [reply AnnoyancesFilter/Cookies/sections/cookies_specific.txt:148]: Are you watching with the desktop application or with the extension?

Comment (BlazDT) [reply AnnoyancesFilter/Cookies/sections/cookies_specific.txt:148]: Android and Windows Chrome Extension.

Comment (Alex-302) [reply AnnoyancesFilter/Cookies/sections/cookies_specific.txt:148]: Same. Elemhide rule is not required with the second rule.

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**